os/drivers/input/touchscreen.c

```diff
@@ -229,7 +229,14 @@ static int touch_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 	}
 
 	switch (cmd) {
-		default:
+		case TSIOC_SUSPEND: {
+			priv->ops->touch_disable(priv);
+		}
+		break;
+
+		case TSIOC_WAKEUP: {
+			priv->ops->touch_enable(priv);
+		}
 		break;
```

Comment:
Sorry but real operations are touch disable and enable. But the commands are SUSPEND and WAKEUP. wakeup and enable look little bit different. Is it impossible to use ENABLE and DISABLE?